packages/react-proptypes/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@stardust-ui/react-proptypes",
+  "description": "Set of custom reusable propTypes for React components.",
```

Comment:
would suggest to capitalize `propTypes`->`PropTypes`, to make it evident that this is the same term that is used in React docs: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
also, we should make it clear in the description that these PropTypes are, in fact, meant to be Stardust specific (not just general-use ones), as currently we have the following provided by the package: `itemShorthandWithKindProp`, `wrapperShorthand`

Reply:
Changed to: `Set of custom reusable PropTypes for React components, some of them are specific for Stardust UI.`